department-of-veterans-affairs/caseflow
# Ki/APPEALS-29424 - Intake Form Step 3: Confirmation - About the Correspondence

Resolves [APPEALS-29424](https://jira.devops.va.gov/browse/APPEALS-29424)

# Description
Establishes the About the Correspondence section on the third step of the Intake Form process.  Also updates the multi_correspondences logic to include more correspondences, matching correspondence documents, and appeals.

## Acceptance Criteria

## Testing Plan
1. Go to Intake Form
2. Navigate to Step 3
3. Verify that page loads with the correct information:
    1. Review and Confirm Correspondence Title and description (explanation text)
    2. About the Correspondence Section
        1. Portal Entry Date
        2. Source Type
        3. Package Document Type
        4. CM Packet Number
        5. CMP Queue Name
        6. VA DOR
        7. Veteran (First name, last name, file number)
        8. Correspondence Type
        9. Notes
 

# Frontend
## User Facing Changes
### Original Feature Build
![Screenshot 2023-11-24 at 3 18 06 PM](https://github.com/department-of-veterans-affairs/caseflow/assets/108023343/00b55fa9-5620-4bbf-972d-61b4b3f6d58b)

### With Merged Feature Branch Changes
![Screenshot 2023-11-27 at 2 28 41 PM](https://github.com/department-of-veterans-affairs/caseflow/assets/108023343/045f788f-06ed-4ec1-b735-5a5ae87cee54)
